components/proxy/conf/Caddyfile

```diff
@@ -172,6 +172,25 @@ http:// {
 	redir https://{host}{uri} permanent
 }
 
+https://services.{$GITPOD_DOMAIN} {
+	import enable_log
+	import remove_server_header
+	import ssl_configuration
+	import security_headers
+
+	handle /idp/* {
+		uri strip_prefix /idp/
+
+		reverse_proxy public-api-server.{$KUBE_NAMESPACE}.{$KUBE_DOMAIN}:9002
+	}
+
+	handle /apps/* {
```

Comment:
Do we want to make this `webhooks/apps/*`?

Reply:
As we need to stay compatible with the current pathes (there are many configured webhooks) I believe it is not worth changing